### PR TITLE
Fix: excluding service tenant and users which are related to services fr...

### PR DESCRIPTION
**Pull request by toha10**
...om the migration process.

We need to avoid service users migration, f.e. nova, glance, etc.

### CODE REVIEW

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/CloudFerry_PR_checks/204/

File: cloudferrylib/os/identity/keystone.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
 LOG = utl.get_log(__name__)
 
 NOVA_SERVICE = 'nova'
+SERVICES_TENANT_NAME = 'services'
```
Comment (roman-verchikov) [inline cloudferrylib/os/identity/keystone.py]: What if services tenant name is different from 'services'?

**Comment by roman-verchikov** [comment]
1. Change commit message to meet requirements used everywhere:
```
<Short summary>

<Commit description>
```
2. You may need to introduce new variable for service user instead of using hardcoded value

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/9/

**Comment by Wasapl** [comment]
Merged build finished. Test FAILed.   http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/13/

**Comment by Wasapl** [comment]
Merged build finished. Test PASSed.  http://jenkins-cf.vm.mirantis.net:8080/job/cloudferry-unittests-and-style-checks/14/

**Comment by roman-verchikov** [comment]
Please squash two commits into one

**Pull request closed without merging**